src/txn/txn_rollback_to_stable.c

```diff
@@ -230,9 +229,47 @@ static int
 __txn_abort_newer_updates(
     WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t *rollback_timestamp)
 {
+	WT_DECL_RET;
 	WT_PAGE *page;
+	uint32_t read_flags;
+	bool local_read;
+
+	local_read = false;
+	read_flags = WT_READ_WONT_NEED;
+	if (ref->page_las != NULL && ref->page_las->skew_newest &&
```

Comment:
A comment describing why this is necessary would be good. i.e: sweep could cleanup the required history after rollback has completed if the previous reconciliation wrote the most recent version of data.

Reply:
Done.